src/js/mep-player.js

```diff
@@ -52,6 +52,13 @@ function constrainedSeekTo(player, media, targetTime) {
 		defaultSeekForwardInterval: function(media) {
 			return (media.duration * 0.05);
 		},
+		defaultJumpBackwardInterval: function(media) {
+			return (media.duration * 0.05);
+		},
+		// default amount to move forward when forward key is pressed
```

Comment:
this comment should differentiate the option from that of `defaultSeekForwardInterval`, and `defaultJumpBackwardInterval` should have a corresponding comment